### PR TITLE
Add Node planner test and additional checks for planner fields

### PR DESCRIPTION
**Pull request by LucilleH**
## Summary
Add Node planner test and additional checks for planner fields

## How was it tested?
go test ./...

### CODE REVIEW

File: testdata/php/php8.1/plan.json [hunk 1]
```diff
@@ -1,17 +1,25 @@
...
-  ],
-  "install_stage": {
-    "command": "composer install --no-dev --no-ansi"
-  },
-  "build_stage": {},
-  "start_stage": {
-    "command": "php -S 0.0.0.0:8080 -t public"
-  }
-}
+    "php81Packages.composer"
```
Comment (LucilleH) [inline testdata/php/php8.1/plan.json:23]: @mikeland86 I noticed this has changed. Is that expected?

Comment (mikeland86) [reply testdata/php/php8.1/plan.json:23]: yeah, we now support php building.

**Comment by mikeland86** [comment]
ugh, I think we both did the same work?

**Comment by LucilleH** [comment]
> ugh, I think we both did the same work?

That's ok 😄

**Pull request merged**